index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Web: school</title>
+  <link rel="stylesheet" href="./style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div id="menu-section" class="menu-section">
+    <button id="menu-toggle" class="menu-toggle"><i class="fa-solid fa-xmark"></i></button>
+    <ul class="menu-list">
+      <li class="menu-item"><a href="#about">About</a></li>
+      <li class="menu-item"><a href="#about">Program</a></li>
+      <li class="menu-item"><a href="#">Join</a></li>
+      <li class="menu-item"><a href="#">Scholarship</a></li>
+      <li class="menu-item"><a href="#">Upcoming classes</a></li>
+    </ul>
+  </div>
+  <section class="home-page">
+    <header class="nav">
+      <button id="menu-icon" class="toggle"><i class="fa-solid fa-bars"></i></button>
+      <nav class="navbar">
+        <a href="#home">Web School</a>
+        <div class="navbar-container">
+          <ul class="navbar-list">
+          <li class="navbar-item"><a href="#about">About </a></li>
+          <li class="navbar-item"><a href="#about">Program</a></li>
+          <li class="navbar-item"><a href="#home">Join</a></li>
+          <li class="navbar-item"><a href="#Scholarship">Scholarship</a></li>
```

Comment:
in your navigation bar, 😸  you should follow the template design as closely as possible 
your nav bar should have a mini bar to host the links to the social media accounts as shown below.

your design
![image](https://user-images.githubusercontent.com/60612329/219875045-2f04fcfd-1dfa-49f9-9eee-eca5c137c6dc.png)

template design
![image](https://user-images.githubusercontent.com/60612329/219875163-b51eccc9-db96-48b0-ab19-9109ed0233e8.png)